.buildkite/scripts/steps/test/scout_test_run_builder.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import path from 'path';
+import { CiStats } from '#pipeline-utils';
+
+(async () => {
+  try {
+    const scoutConfigsPath = path.resolve(process.cwd(), '.scout', 'scout_playwright_configs.json');
```

Comment:
Do we have access to the scout-info package from that pipeline script? If so, it would be good to replace this hard-coded path with the imported value from the info package.

Reply:
```

/opt/buildkite-agent/.kibana/node/20.18.2/lib/node_modules/ts-node/src/index.ts:859
--
  | 2025-03-11 14:09:53 UTC | return new TSError(diagnosticText, diagnosticCodes, diagnostics);
  | 2025-03-11 14:09:53 UTC | ^
  | 2025-03-11 14:09:53 UTC | TSError: ⨯ Unable to compile TypeScript:
  | 2025-03-11 14:09:53 UTC | src/platform/packages/private/kbn-scout-info/src/paths.ts:10:18 - error TS2307: Cannot find module 'node:path' or its corresponding type declarations.
  | 2025-03-11 14:09:53 UTC |  
  | 2025-03-11 14:09:53 UTC | 10 import path from 'node:path';
```